src/scripts/main.js

```diff
@@ -20,6 +20,29 @@ const food = {
 
 const tree = document.querySelector('#tree');
 
+function createTree(element, data) {
+  const list = document.createElement('ul');
+
+  Object.keys(data).forEach((key) => {
+    const li = document.createElement('li');
+
+    li.textContent = key;
+    list.appendChild(li);
+
+    if (Object.keys(data[key]).length > 0) {
+      createTree(li, data[key]);
+    }
+  });
+
+  element.appendChild(list);
+
+  return list;
+}
+
+createTree(tree, food);
+
+tree = document.querySelector('#tree');
+
 function createTree(element, data) {
   // WRITE YOUR CODE HERE
 }
```

Comment:
The `createTree` function is already defined on lines 23-40. This redefinition is unnecessary and should be removed to avoid confusion and potential errors.